Marlin/src/gcode/bedlevel/M420.cpp

```diff
@@ -156,18 +156,30 @@ void GcodeSuite::M420() {
 
             // Get the sum and average of all mesh values
             float mesh_sum = 0;
-            GRID_LOOP(x, y) mesh_sum += bedlevel.z_values[x][y];
-            const float zmean = mesh_sum / float(GRID_MAX_POINTS);
-
+            #if ENABLED(AUTO_BED_LEVELING_UBL)
```

Comment:
This is inside the `#else` for `#if ENABLED(AUTO_BED_LEVELING_UBL)` and therefore this can never be true.

Reply:
no problem. this was a hurry shot to fix compilation. do whatever you think is right ;)